packages/core/src/package-utils/bundle.ts

```diff
@@ -43,26 +43,22 @@ export const createBundle = async (
         );
 
         // internal package resolved by plugins/vite/internal-package-resolution.ts
-        if (srcPath.startsWith('../')) return;
-
-        assert(
-          !srcPath.startsWith('./'),
-          `relative path ${srcPath} should not start with './'`,
-        );
+        if (srcPath.startsWith('../')) {
+          logger.debug(`Internal package ${srcPath}`);
+          return;
+        }
 
-        if (
-          isVanillaFile(id) ||
-          getModuleInfo(id)?.importers.some(isVanillaFile)
```

Comment:
Removing this means modules which import Vanilla files are no longer placed in `stylesDir` as a separate file. Rollup may decide to merge them with other chunks and I am ok with that.

Reply:
Reverting this change after testing in apps consuming Braid.